Makefile

```diff
@@ -1,3 +1,16 @@
+# Variables:
+# V=1 for verbose output.
+# BINNAME=step to change the name of the executable
+# PREFIX=bin/ to change the build output path
```

Comment:
This is weird because we're using this bellow `$(PREFIX)bin/$(BINNAME).`

Shoudl we set `PREFIX?=bin/` and use `$(PREFIX)$(BINNAME)` or perhaps better `PREFIX?=bin` and `$(PREFIX)/$(BINNAME)`?

Reply:
Good call, I went with `PREFIX?=bin` and `DESTDIR?=/usr/local/bin`.